### ISSUE TITLE
Add "Has_edge_between_vertices" for directed graph

### ISSUE THREAD

**Issue by Lumphie** [richelbilderbeek/BoostGraphTutorial#44]
The tutorial did not mention anything about edges between vertices for directed graphs.

If you use the function `has_edge_between_vertices(vd_1, vd_2, g)` it can give a different than `has_edge_between_vertices(vd_2, vd_1, g)` output in a directed graph, depending on what direction the edge has.

**Comment by richelbilderbeek** [comment]
This is correct behavior. It should be mentioned in the text, or more functions with directed graphs should be added. 

Feel encouraged to submit a PR :+1:
